FirebaseSegmentation/Tests/Sample/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be checked in. The project should be invalidated and a new one encoded instead.

Reply:
removed